Citavi 5 XML.js

```diff
@@ -449,12 +473,45 @@ async function doImport() {
 	const totalProgress = references.length + tasks.length + categories.length;
 	const progress = { total: totalProgress * 2, current: 0 };
 
-	await importItems({ references, doc, citaviVersion, rememberTags, itemIdList, progress, unfinishedReferences });
+	await importItems({ references, doc, citaviVersion, rememberTags, rememberCustomFields, itemIdList, progress, unfinishedReferences });
 	await importUnfinished({ doc, itemIdList, unfinishedReferences, progress });
 	await importTasks({ tasks, progress });
+	await importStandalonNotes(doc, rememberTags, progress);
 	importCategories({ categories, doc, progress });
 }
 
+function extractNote(doc, noteXML, rememberTags) {
```

Comment:
One major drawback when import data from Citavi to Zotero is loosing all knowledge-items with images ("Bildzitat"). As far as I am aware, it is currently not possible to attach an image to a note, which results in many notes without much content after import. To simplify manual fixing afterward, I suggest tagging the note and include the filename into the note, as follows. The result could look like
![grafik](https://github.com/user-attachments/assets/e21d739b-4fe2-4a83-a0ee-66d61a293466)